### DISCUSSION TITLE
Chart/Labels Sharpness

### DISCUSSION THREAD

**Discussion by devtwo7** [Live-Charts/LiveCharts2#387] [Q&A]
Using a CartesianChart, the chart lines and label text are rather blurry. I assume this is an issue with SkiaSharp? I don't know much about Skia in particular, a search for a solution to this leads me to two different properties that could affect this. "SnapToDevicePixels" and "UseLayoutRounding" (link to my search result: [https://github.com/mono/SkiaSharp/issues/382](url)). It seems neither of these helped me out. Attached is a snapshot of part of my chart where you can see the blurriness of both the text and lines. This occurs both at full scale and zoomed, but it is a pretty large chart so zooming exaggerates the issue a bit.
![Capture](https://user-images.githubusercontent.com/91838893/152862229-b964d255-e520-41d2-b0a4-b69a2f3bff74.PNG)
 Any help?

**Comment by RamenJamen** [comment]
Any update on this?
